kotlin-grpc/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+dependencies {
+    implementation(project(":kotlin"))
+    implementation(project(":grpc"))
+    implementation(project(":junit5"))
```

Comment:
`junit5` dependency is added by default.
https://github.com/line/armeria/blob/b141ecccb7b8686973ca9a8edff8248ea7d05ea0/build.gradle#L123-L131
```suggestion
```

Reply:
Ref: https://github.com/line/armeria/pull/4724#discussion_r1127589412